src/foam/nanos/ticket/Ticket.js

```diff
@@ -73,6 +73,14 @@ foam.CLASS({
     'createdFor'
   ],
 
+  searchColumns: [
+    'id',
+    'status',
+    'comment',
```

Comment:
This won't work as expected, as noted in discussions.

Reply:
right sorry
updated